packages/plugins/blocks/blocks-antd/src/blocks/DateRangeSelector/DateRangeSelector.js

```diff
@@ -51,7 +51,7 @@ const DateRangeSelector = ({
       validation={validation}
       required={required}
       content={{
-        content: () => (
+        content: (icon) => (
```

Comment:
Can we add an TODO here to say: Content area should pass down named args.

Reply:
Actually, let's change these to `({ icon }) => ` and call with `({ icon })`, please.

Reply:
Change made in 832f68c671e23c30f960792c8dd91164c6742f6a